apps/expo/src/app/privacy-policy/index.tsx

```diff
@@ -0,0 +1,225 @@
+import React, { useState } from 'react'
+import { Switch, Text, View } from 'react-native'
+import { ScrollView } from 'react-native-gesture-handler'
+import { SafeAreaView } from 'react-native-safe-area-context'
+
+import Label from '~/components/ui/label/Label'
+
+const tailwindClasses = {
+  mainTile: 'm-4 p-4 rounded-lg bg-bb-dark-gray',
+  mainTileTitle: 'text-2xl text-slate-200',
+  mainTileItem: 'flex-row',
+  navigationListItemLabel: 'flex-auto m-2 self-center text-slate-200',
+  navigationListItemChevron: 'self-center text-slate-200',
+}
+
+const PrivacyPolicy = () => {
+  const [isEnabled, setIsEnabled] = useState(false)
+  const toggleSwitch = () => setIsEnabled((previousState) => !previousState)
+
+  const privacyToggleSwitch = async () => {}
+  // The privacy policy was AI generated via a prompt to chatGPT, the link to the prompt is below
+  // https://chat.openai.com/share/edbfaba8-f225-4168-8d86-c8263ed9ddc5
+  return (
+    <SafeAreaView
+      className='bg-bb-slate-100 flex-1'
+      style={{ backgroundColor: '#1e1e1e', flex: 1 }}
+    >
+      <ScrollView>
+        <View>
+          <Text
+            className='text-2xl text-slate-200'
+            style={{
+              fontSize: 40,
+              textAlign: 'center',
+              paddingHorizontal: 10,
+              fontWeight: 'bold',
+            }}
+          >
+            Privacy Policy
+          </Text>
+
+          <View style={{ paddingHorizontal: 30, flexDirection: 'row' }}>
+            <Label text='Legal Document' textColor='#CFCFCF' backgroundColor='#48476D' />
+            <Text> </Text>
+            <Label text='Human-Friendly' textColor='#CFCFCF' backgroundColor='#48476D' />
```

Comment:
have it something else than `human-friendly`.

and `legal documents`, sounds a bit erm...We might have something else?

Reply:
Changed to `Data Collection` and `Third-Party Integration`